main.go

```diff
@@ -3,11 +3,11 @@ package main
 import (
 	"crypto/tls"
 	"crypto/x509"
```

Comment:
It would also be ideal if you published the code that generates the certificates so the program can be tested.

Reply:
perhaps I should do this in another pr? It would be relatively unproblematic in terms of merge conflicts.